### ISSUE TITLE
Get application and remove application

### ISSUE THREAD

**Issue by chrislovecnm** [jenkins-x/jx#8153]
Once I have created an application how do I remove the application?  When I run

```bash
$ jx application get
APPLICATION
```

I returns nothing, but I have builds that run and I am generating previews.  So I do not know how to run `jx application delete`.

**Comment by ankitm123** [comment]
May be an issue with different versions of go, jx-application still uses an old version of go, I will update it soon.

**Comment by falsedlah** [comment]
I have looked into this issue yesterday and I found the problem at least in my case. In fact, if the app repo's git name is different than helm chart's name it will be filtered out. 
https://github.com/jenkins-x-plugins/jx-application/blob/main/pkg/applications/applications.go#L185

In my case, they are different so only the repos that have the same repo & chart name, they can show. The fix ideally ignore the repo name and rely on the app name itself but I don't know if this something against jx concept?
